Homework for Session1

StefanescuEduard opened the pull request.

Zestysian committed:
Homework for Session1

CosminBarbus commented:
Ok, I will take a look and review the code

CosminBarbus commented on HelloWorld/HelloWorld/Cat.cs:
```diff
@@ -0,0 +1,20 @@
+{
+    class Cat : Animal
+    {
+        public override string Name()
```
How do you ensure that the Name and Noise methods are implemented?

Zestysian committed:
Changes in Animal class

I understand the review. First time, I was thinking to put an if statement when the noises and animals are displayed, to check if the string is empty or not. But then, I rembered, that an abstract method is forced to be implemented in a derived class.
